### PR TITLE
 Get rid of the vendor directory, part 1

### PR DESCRIPTION
**Pull request by gesa**
(first TODO in #246)

### CODE REVIEW

File: components/blockquote.jsx [hunk 1]
```diff
@@ -15,7 +15,6 @@ var Blockquote = React.createClass({
     return (
       <blockquote className={this.props.className}>
         <figure>
-          <img src={this.props.imgSrc} alt={this.props.imgAlt} width="148" />
```
Comment (gesa) [inline components/blockquote.jsx]: this just fixes a regression i accidentally introduced in #237

File: less/styles.less [hunk 2]
```diff
@@ -27,14 +27,14 @@ body {
...
     min-height: 100vh;
-    .flex-block();
-    .flex-direction(column);
   }
 
   .page.container-fluid > .row {
-    .flex-grow(1);
-    .flex-block();
+    display: flex;
+    flex-grow:1;
```
Comment (alicoding) [inline less/styles.less]: spacing :) (sorry about nit picking now :P )

**Comment by toolness** [comment]
Niice! Does this also get rid of `vendor/prefixer.less`?

File: less/styles.less (hunk 2 above)
Comment (gesa) [inline less/styles.less]: it's a good nit! thank you!

File: less/styles.less [hunk 3]
```diff
@@ -56,10 +56,10 @@ body {
   background-size: cover;
   height: 45rem;
   color: white;
+  display: flex;
+  flex-direction: column;
+  justify-content:center;
```
Comment (alicoding) [inline less/styles.less]: I'm not sure how much we care about formatting here if it's a thing then spacing otherwise ignore me

**Comment by alicoding** [comment]
yeah was about to ask about the same as @toolness what we are removing here?

**Comment by gesa** [comment]
Vendor prefixer. I forgot to actually delete the file :grimacing: 

Still working on moving compiled bootstrap from `node_modules` to dist, that'll be a different PR.

**Comment by alicoding** [comment]
You also need to remove `@import "../vendor/prefixer.less";` from `styles.less` too @gesa

**Comment by gesa** [comment]
I don't know *what* you're talking about @alicoding :wink: (sorry i was moving too fast there)

**Comment by toolness** [comment]
Cool, I like the idea of splitting the issues into two separate PRs.

**Comment by alicoding** [comment]
@gesa hahaha

**Comment by alicoding** [comment]
r+ from me as long as travis is happy

**Pull request merged**